cloudinit/cmd/devel/hotplug_hook.py

```diff
@@ -133,6 +149,14 @@ def device_detected(self) -> bool:
 }
 
 
+def is_enabled(hotplug_init, subsystem):
+    scope = SUBSYSTEM_PROPERTES_MAP[subsystem][1]
```

Comment:
run try/except hotplug_init.fetch(existing="trust") here?

Reply:
I don't think we want it here. I applied the try/catch to the other place you suggested, but I don't think we always want to print and exit 1 when we get that exception. E.g., on line 174, if we have a problem there, I want it written the logs. I'm ok with the exception just falling through at that point.